### PR TITLE
Fixes revolvers for real this time (Plus some sounds!)

### PR DESCRIPTION
**Pull request by PopNotes**
:cl: PopNotes
fix: The revolver changes should work now.
soundadd: New sounds for inserting bullets into guns/magazines AND new sounds for using speedloaders!
soundadd: New sound for the detective's revolver. Become the cowboy.
soundadd: New sound for the double-barrel shotgun. Meaty.
/:cl:

### CODE REVIEW

**Comment by PopNotes** [comment]
hippiestation.dme only _looks_ fucky because of an outdated master, there shouldn't be any issues. [](https://www.diffchecker.com/S9ItcQqS) If you want to look for yourself.

**Comment by PopNotes** [comment]
so is anyone gonna take a look at this or

File: hippiestation.dme [hunk 1]
```diff
@@ -1998,10 +2002,22 @@
 #include "code\modules\research\xenobiology\xenobio_camera.dm"
 #include "code\modules\research\xenobiology\xenobiology.dm"
 #include "code\modules\ruins\lavaland_ruin_code.dm"
+#include "code\modules\ruins\lavalandruin_code\sloth.dm"
+#include "code\modules\ruins\lavalandruin_code\surface.dm"
 #include "code\modules\ruins\objects_and_mobs\ash_walker_den.dm"
 #include "code\modules\ruins\objects_and_mobs\necropolis_gate.dm"
 #include "code\modules\ruins\objects_and_mobs\ruin_mapping_aids.dm"
 #include "code\modules\ruins\objects_and_mobs\sin_ruins.dm"
+#include "code\modules\ruins\spaceruin_code\asteroid4.dm"
```
Comment (GuyonBroadway) [inline hippiestation.dme]: So what's with all these lines?

Just do us a favour and delete any extra lines via the web editor that aren't supposed to be added by your PR.

File: hippiestation.dme [hunk 2]
```diff
@@ -976,7 +976,6 @@
 #include "code\game\turfs\simulated\floor\reinf_floor.dm"
 #include "code\game\turfs\simulated\floor\plating\asteroid.dm"
 #include "code\game\turfs\simulated\floor\plating\dirt.dm"
-#include "code\game\turfs\simulated\floor\plating\lava.dm"
```
Comment (GuyonBroadway) [inline hippiestation.dme]: End of rine.... land

File: hippiestation.dme [hunk 3]
```diff
@@ -1905,7 +1904,6 @@
 #include "code\modules\projectiles\guns\mounted.dm"
 #include "code\modules\projectiles\guns\syringe_gun.dm"
 #include "code\modules\projectiles\guns\ballistic\automatic.dm"
-#include "code\modules\projectiles\guns\ballistic\bow.dm"
```
Comment (GuyonBroadway) [inline hippiestation.dme]: rine ends here

**Comment by GuyonBroadway** [comment]
build this

**Comment by GuyonBroadway** [comment]
hello

**Comment by GuyonBroadway** [comment]
build this

**Comment by PopNotes** [comment]
henlo

**Comment by PopNotes** [comment]
yeah boy (yeah boy)

**Comment by GuyonBroadway** [comment]
Nooooooice

**Comment by PopNotes** [comment]
simply eric

**Pull request merged**